pkg/file/redundancy/level.go

```diff
@@ -135,3 +150,19 @@ var encParanoidEt = newErasureTable(
 		55, 51, 48, 44, 39, 35, 30, 24,
 	},
 )
+
```

Comment:
These changes expose a lot of convoluted internals to public which is a sign that they are not at the right place. 
I think this should remain as per the original, honestly.